### DISCUSSION TITLE
Backup: creating a snapshot of a database

### DISCUSSION THREAD

**Discussion by Yuzal** [ravendb/ravendb#15346] [Q&A] [answered]
Scenario:
We have a database which is usually used by a single user for demonstrations and presentations which can be dumped anytime for a new database if needed.
I would like to implement him an option to make a "snapshot" of the current database anytime he wants and to reset the database to the latest "snapshot" (for example after a demonstration he wants to get rid of anything he added during the demonstration)

one option i had in mind would be to set up a simple backup server =>
Snapshot: i delete the old database and create a new database and export the current active one.
Reset: i delete the current database, create a new one and import the backup database.

Are there any other options with ravendb?

**Comment by ayende** [comment, marked as answer]
Import & export are the easiest ways to handle that.
Alternatively, you can setup two databases:

* Snapshot
* Active

Setup external replication from Snapshot --> Active

Then you can modify the data in Active as you see fit.
When you want to go back, you delete the Active database and create it again.
That database will be filled with the Snapshot details from the other database.
